packages/components/src/components/table/table-header.tsx

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 
-const Header = ({ children, className }) => (
+type THeader = {
+    children: React.ReactNode;
+    className?: string;
+};
+
+const Header = ({ children, className }: THeader) => (
```

Comment:
```suggestion
const Header = ({ children, className }: React.PropsWithChildren<THeader>) => (
```
```
type THeader = {
    className?: string;
};
```

Reply:
thanks for this comment. in other cases it would be great, but here children can’t be optional. in PropsWithChildren children is optional.